docs/troubleshooting/semgrep-app.md

```diff
@@ -173,18 +173,16 @@ Check the log output for any hints about what the issue is.
   ![Screenshot of GitHub's Actions tab with workflows disabled](/img/github-workflows-disabled.png)
   - Enable workflows by clicking **I understand my workflows, go ahead and enable them** to allow Semgrep to scan.
 
-### If a project reports the last scan 'Never finished'
+### If a project reports a scan 'Never finished'
 
-This status means that your CI jobs start and authenticate correctly, but fail before completion.
+Most often, this status means that your CI jobs start and authenticate correctly, but fail or are canceled before completion. Check your CI provider (such as GitHub Actions) for the log output of the latest Semgrep job execution. In most cases, you will see an error message with detailed instructions on what to do.
 
-Check your CI provider (such as GitHub Actions) for the log output of the latest Semgrep job execution. In most cases you will see an error message with detailed instructions on what to do.
+Sometimes, this status may be shown when the scan has been running for a long time (more than an hour) and is still in progress. Scans that eventually produce results will always be accepted by Semgrep AppSec Platform, even if this message is shown.
```

Comment:
I appreciate the intent of including the specific timeout, but I don't trust us to remember to update the docs if we change the timeout. 😅 And while "always" _should_ be true, in practice it's not, so the implied "this is what's meant to happen" feels more right than emphasizing it.

```suggestion
Sometimes, this status may be shown when the scan has been running for a long time and is still in progress. Scans that eventually produce results will be accepted by Semgrep AppSec Platform, even if this message is shown.
```

Reply:
Yeah I agree about always, but I'm not sure I agree about the timeout. I don't think we are likely to shorten it, so I feel okay about more than an hour.

Reply:
I committed that half of the suggestion, appreciate you taking a look!